code-reviews.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Code Reviews at OfficeLuv
+layout: default
+permalink: '/code-reviews'
+---
+
+## Why Do Code Reviews?
+
+Doing code reviews:
+
+- shares knowledge of the codebase and business
+- catches bugs automated testing may not catch
```

Comment:
```suggestion
- catches bugs automated testing may not catch
- uncovers optimizations and opportunities for collaboration that may otherwise stay siloed
```